[2018-11-18 05:12:15 +0000] [10139] [INFO] Listening at: http://127.0.0.1:8000 (10139)
[2018-11-18 05:12:15 +0000] [10139] [INFO] Using worker: sync
[2018-11-18 05:12:15 +0000] [10143] [INFO] Booting worker with pid: 10143
/opt/anaconda2/lib/python2.7/site-packages/h5py/__init__.py:36: FutureWarning: Conversion of the second argument of issubdtype from `float` to `np.floating` is deprecated. In future, it will be treated as `np.float64 == np.dtype(float).type`.
  from ._conv import register_converters as _register_converters
2018-11-18 05:12:16.557261: I tensorflow/core/platform/cpu_feature_guard.cc:141] Your CPU supports instructions that this TensorFlow binary was not compiled to use: SSE4.1 SSE4.2 AVX AVX2 FMA
2018-11-18 05:12:16.558535: I tensorflow/core/common_runtime/process_util.cc:69] Creating new thread pool with default inter op setting: 2. Tune using inter_op_parallelism_threads for best performance.
[2018-11-18 05:27:43 +0000] [10139] [INFO] Handling signal: term
[2018-11-18 05:27:43 +0000] [10143] [INFO] Worker exiting (pid: 10143)
looking for food in files/CMTKPBF.png
looking for food in files/CMTKPBF.png
looking for food in files/NU91NXP.png
looking for food in files/NU91NXP.png
[2018-11-18 05:27:43 +0000] [10139] [INFO] Shutting down: Master
[2018-11-18 05:27:45 +0000] [10271] [INFO] Starting gunicorn 19.9.0
[2018-11-18 05:27:45 +0000] [10271] [INFO] Listening at: http://127.0.0.1:8000 (10271)
[2018-11-18 05:27:45 +0000] [10271] [INFO] Using worker: sync
[2018-11-18 05:27:45 +0000] [10275] [INFO] Booting worker with pid: 10275
/opt/anaconda2/lib/python2.7/site-packages/h5py/__init__.py:36: FutureWarning: Conversion of the second argument of issubdtype from `float` to `np.floating` is deprecated. In future, it will be treated as `np.float64 == np.dtype(float).type`.
  from ._conv import register_converters as _register_converters
2018-11-18 05:27:46.496315: I tensorflow/core/platform/cpu_feature_guard.cc:141] Your CPU supports instructions that this TensorFlow binary was not compiled to use: SSE4.1 SSE4.2 AVX AVX2 FMA
2018-11-18 05:27:46.497550: I tensorflow/core/common_runtime/process_util.cc:69] Creating new thread pool with default inter op setting: 2. Tune using inter_op_parallelism_threads for best performance.
[2018-11-18 05:30:36 +0000] [10271] [INFO] Handling signal: term
[2018-11-18 05:30:36 +0000] [10275] [INFO] Worker exiting (pid: 10275)
[2018-11-18 05:30:37 +0000] [10271] [INFO] Shutting down: Master
[2018-11-18 05:30:38 +0000] [10432] [INFO] Starting gunicorn 19.9.0
[2018-11-18 05:30:38 +0000] [10432] [INFO] Listening at: http://127.0.0.1:8000 (10432)
[2018-11-18 05:30:38 +0000] [10432] [INFO] Using worker: sync
[2018-11-18 05:30:38 +0000] [10436] [INFO] Booting worker with pid: 10436
/opt/anaconda2/lib/python2.7/site-packages/h5py/__init__.py:36: FutureWarning: Conversion of the second argument of issubdtype from `float` to `np.floating` is deprecated. In future, it will be treated as `np.float64 == np.dtype(float).type`.
  from ._conv import register_converters as _register_converters
2018-11-18 05:30:39.727134: I tensorflow/core/platform/cpu_feature_guard.cc:141] Your CPU supports instructions that this TensorFlow binary was not compiled to use: SSE4.1 SSE4.2 AVX AVX2 FMA
2018-11-18 05:30:39.728416: I tensorflow/core/common_runtime/process_util.cc:69] Creating new thread pool with default inter op setting: 2. Tune using inter_op_parallelism_threads for best performance.
[2018-11-18 05:39:05 +0000] [10432] [INFO] Handling signal: term
[2018-11-18 05:39:05 +0000] [10436] [INFO] Worker exiting (pid: 10436)
[2018-11-18 05:39:05 +0000] [10432] [INFO] Shutting down: Master
[2018-11-18 05:39:06 +0000] [10586] [INFO] Starting gunicorn 19.9.0
[2018-11-18 05:39:06 +0000] [10586] [INFO] Listening at: http://127.0.0.1:8000 (10586)
[2018-11-18 05:39:06 +0000] [10586] [INFO] Using worker: sync
[2018-11-18 05:39:06 +0000] [10590] [INFO] Booting worker with pid: 10590
/opt/anaconda2/lib/python2.7/site-packages/h5py/__init__.py:36: FutureWarning: Conversion of the second argument of issubdtype from `float` to `np.floating` is deprecated. In future, it will be treated as `np.float64 == np.dtype(float).type`.
  from ._conv import register_converters as _register_converters
2018-11-18 05:39:07.875440: I tensorflow/core/platform/cpu_feature_guard.cc:141] Your CPU supports instructions that this TensorFlow binary was not compiled to use: SSE4.1 SSE4.2 AVX AVX2 FMA
2018-11-18 05:39:07.876656: I tensorflow/core/common_runtime/process_util.cc:69] Creating new thread pool with default inter op setting: 2. Tune using inter_op_parallelism_threads for best performance.
[2018-11-18 05:39:20,371] ERROR in app: Exception on /upload [GET]
Traceback (most recent call last):
  File "/home/ubuntu/.local/lib/python2.7/site-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/ubuntu/.local/lib/python2.7/site-packages/flask/app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/ubuntu/.local/lib/python2.7/site-packages/flask/app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/ubuntu/.local/lib/python2.7/site-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/ubuntu/.local/lib/python2.7/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/ubuntu/SeeFood-Server/index.py", line 43, in upload_file
    images = filesFromDB(minRow, maxRow)
  File "/home/ubuntu/SeeFood-Server/index.py", line 138, in filesFromDB
    (minRow, maxRow)
TypeError: %d format: a number is required, not NoneType
[2018-11-18 05:39:26,633] ERROR in app: Exception on /upload [GET]
Traceback (most recent call last):
  File "/home/ubuntu/.local/lib/python2.7/site-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/ubuntu/.local/lib/python2.7/site-packages/flask/app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/ubuntu/.local/lib/python2.7/site-packages/flask/app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/ubuntu/.local/lib/python2.7/site-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/ubuntu/.local/lib/python2.7/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/ubuntu/SeeFood-Server/index.py", line 43, in upload_file
    images = filesFromDB(minRow, maxRow)
  File "/home/ubuntu/SeeFood-Server/index.py", line 138, in filesFromDB
    (minRow, maxRow)
TypeError: %d format: a number is required, not NoneType
[2018-11-18 05:41:39 +0000] [10586] [INFO] Handling signal: term
[2018-11-18 05:41:39 +0000] [10590] [INFO] Worker exiting (pid: 10590)
[2018-11-18 05:41:39 +0000] [10586] [INFO] Shutting down: Master
[2018-11-18 05:41:41 +0000] [10695] [INFO] Starting gunicorn 19.9.0
[2018-11-18 05:41:41 +0000] [10695] [INFO] Listening at: http://127.0.0.1:8000 (10695)
[2018-11-18 05:41:41 +0000] [10695] [INFO] Using worker: sync
[2018-11-18 05:41:41 +0000] [10699] [INFO] Booting worker with pid: 10699
/opt/anaconda2/lib/python2.7/site-packages/h5py/__init__.py:36: FutureWarning: Conversion of the second argument of issubdtype from `float` to `np.floating` is deprecated. In future, it will be treated as `np.float64 == np.dtype(float).type`.
  from ._conv import register_converters as _register_converters
2018-11-18 05:41:42.459862: I tensorflow/core/platform/cpu_feature_guard.cc:141] Your CPU supports instructions that this TensorFlow binary was not compiled to use: SSE4.1 SSE4.2 AVX AVX2 FMA
2018-11-18 05:41:42.461072: I tensorflow/core/common_runtime/process_util.cc:69] Creating new thread pool with default inter op setting: 2. Tune using inter_op_parallelism_threads for best performance.
[2018-11-18 05:41:45,758] ERROR in app: Exception on /upload [GET]
Traceback (most recent call last):
  File "/home/ubuntu/.local/lib/python2.7/site-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/ubuntu/.local/lib/python2.7/site-packages/flask/app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/ubuntu/.local/lib/python2.7/site-packages/flask/app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/ubuntu/.local/lib/python2.7/site-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/ubuntu/.local/lib/python2.7/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/ubuntu/SeeFood-Server/index.py", line 53, in upload_file
    zf.write(individualFile[1], basename(individualFile[1]))
IndexError: string index out of range
[2018-11-18 05:42:55 +0000] [10695] [INFO] Handling signal: term
[2018-11-18 05:42:55 +0000] [10699] [INFO] Worker exiting (pid: 10699)
(1064, "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ''0','50000'' at line 1")
[2018-11-18 05:42:55 +0000] [10695] [INFO] Shutting down: Master
[2018-11-18 05:42:57 +0000] [10804] [INFO] Starting gunicorn 19.9.0
[2018-11-18 05:42:57 +0000] [10804] [INFO] Listening at: http://127.0.0.1:8000 (10804)
[2018-11-18 05:42:57 +0000] [10804] [INFO] Using worker: sync
[2018-11-18 05:42:57 +0000] [10808] [INFO] Booting worker with pid: 10808
/opt/anaconda2/lib/python2.7/site-packages/h5py/__init__.py:36: FutureWarning: Conversion of the second argument of issubdtype from `float` to `np.floating` is deprecated. In future, it will be treated as `np.float64 == np.dtype(float).type`.
  from ._conv import register_converters as _register_converters
2018-11-18 05:42:58.146036: I tensorflow/core/platform/cpu_feature_guard.cc:141] Your CPU supports instructions that this TensorFlow binary was not compiled to use: SSE4.1 SSE4.2 AVX AVX2 FMA
2018-11-18 05:42:58.147842: I tensorflow/core/common_runtime/process_util.cc:69] Creating new thread pool with default inter op setting: 2. Tune using inter_op_parallelism_threads for best performance.
[2018-11-18 05:43:01,568] ERROR in app: Exception on /upload [GET]
Traceback (most recent call last):
  File "/home/ubuntu/.local/lib/python2.7/site-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/ubuntu/.local/lib/python2.7/site-packages/flask/app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/ubuntu/.local/lib/python2.7/site-packages/flask/app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/ubuntu/.local/lib/python2.7/site-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/ubuntu/.local/lib/python2.7/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/ubuntu/SeeFood-Server/index.py", line 53, in upload_file
    zf.write(individualFile[1], basename(individualFile[1]))
IndexError: string index out of range
[2018-11-18 05:44:08 +0000] [10804] [INFO] Handling signal: term
[2018-11-18 05:44:08 +0000] [10808] [INFO] Worker exiting (pid: 10808)
(1064, "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ''0','50000'' at line 1")
[2018-11-18 05:44:09 +0000] [10804] [INFO] Shutting down: Master
[2018-11-18 05:44:10 +0000] [10914] [INFO] Starting gunicorn 19.9.0
[2018-11-18 05:44:10 +0000] [10914] [INFO] Listening at: http://127.0.0.1:8000 (10914)
[2018-11-18 05:44:10 +0000] [10914] [INFO] Using worker: sync
[2018-11-18 05:44:10 +0000] [10918] [INFO] Booting worker with pid: 10918
/opt/anaconda2/lib/python2.7/site-packages/h5py/__init__.py:36: FutureWarning: Conversion of the second argument of issubdtype from `float` to `np.floating` is deprecated. In future, it will be treated as `np.float64 == np.dtype(float).type`.
  from ._conv import register_converters as _register_converters
2018-11-18 05:44:11.484097: I tensorflow/core/platform/cpu_feature_guard.cc:141] Your CPU supports instructions that this TensorFlow binary was not compiled to use: SSE4.1 SSE4.2 AVX AVX2 FMA
2018-11-18 05:44:11.485361: I tensorflow/core/common_runtime/process_util.cc:69] Creating new thread pool with default inter op setting: 2. Tune using inter_op_parallelism_threads for best performance.
[2018-11-18 05:44:14,812] ERROR in app: Exception on /upload [GET]
Traceback (most recent call last):
  File "/home/ubuntu/.local/lib/python2.7/site-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/ubuntu/.local/lib/python2.7/site-packages/flask/app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/ubuntu/.local/lib/python2.7/site-packages/flask/app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/ubuntu/.local/lib/python2.7/site-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/ubuntu/.local/lib/python2.7/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/ubuntu/SeeFood-Server/index.py", line 55, in upload_file
    zf.write(individualFile[1], basename(individualFile[1]))
IndexError: string index out of range
[2018-11-18 05:47:04 +0000] [10914] [INFO] Handling signal: term
[2018-11-18 05:47:04 +0000] [10918] [INFO] Worker exiting (pid: 10918)
None
(1064, "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ''0','50000'' at line 1")
filesFromDB Failed
[2018-11-18 05:47:05 +0000] [10914] [INFO] Shutting down: Master
[2018-11-18 05:47:06 +0000] [11023] [INFO] Starting gunicorn 19.9.0
[2018-11-18 05:47:06 +0000] [11023] [INFO] Listening at: http://127.0.0.1:8000 (11023)
[2018-11-18 05:47:06 +0000] [11023] [INFO] Using worker: sync
[2018-11-18 05:47:06 +0000] [11029] [INFO] Booting worker with pid: 11029
/opt/anaconda2/lib/python2.7/site-packages/h5py/__init__.py:36: FutureWarning: Conversion of the second argument of issubdtype from `float` to `np.floating` is deprecated. In future, it will be treated as `np.float64 == np.dtype(float).type`.
  from ._conv import register_converters as _register_converters
2018-11-18 05:47:07.492841: I tensorflow/core/platform/cpu_feature_guard.cc:141] Your CPU supports instructions that this TensorFlow binary was not compiled to use: SSE4.1 SSE4.2 AVX AVX2 FMA
2018-11-18 05:47:07.494175: I tensorflow/core/common_runtime/process_util.cc:69] Creating new thread pool with default inter op setting: 2. Tune using inter_op_parallelism_threads for best performance.
[2018-11-18 05:48:37,417] ERROR in app: Exception on /upload [GET]
Traceback (most recent call last):
  File "/home/ubuntu/.local/lib/python2.7/site-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/ubuntu/.local/lib/python2.7/site-packages/flask/app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/ubuntu/.local/lib/python2.7/site-packages/flask/app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/ubuntu/.local/lib/python2.7/site-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/ubuntu/.local/lib/python2.7/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/ubuntu/SeeFood-Server/index.py", line 46, in upload_file
    images = filesFromDB(minRow, maxRow)
  File "/home/ubuntu/SeeFood-Server/index.py", line 144, in filesFromDB
    (minRow, maxRow)
TypeError: %d format: a number is required, not unicode
[2018-11-18 05:50:54 +0000] [11023] [INFO] Handling signal: term
[2018-11-18 05:50:54 +0000] [11029] [INFO] Worker exiting (pid: 11029)
None
[['loaded_fries', 'files/CMTKPBF.png', 2.62416, -0.559394], ['loaded_fries', 'files/NU91NXP.png', 2.62416, -0.559394]]
1
[2018-11-18 05:50:54 +0000] [11023] [INFO] Shutting down: Master
[2018-11-18 05:50:55 +0000] [11141] [INFO] Starting gunicorn 19.9.0
[2018-11-18 05:50:55 +0000] [11141] [INFO] Listening at: http://127.0.0.1:8000 (11141)
[2018-11-18 05:50:55 +0000] [11141] [INFO] Using worker: sync
[2018-11-18 05:50:55 +0000] [11145] [INFO] Booting worker with pid: 11145
/opt/anaconda2/lib/python2.7/site-packages/h5py/__init__.py:36: FutureWarning: Conversion of the second argument of issubdtype from `float` to `np.floating` is deprecated. In future, it will be treated as `np.float64 == np.dtype(float).type`.
  from ._conv import register_converters as _register_converters
2018-11-18 05:50:56.796346: I tensorflow/core/platform/cpu_feature_guard.cc:141] Your CPU supports instructions that this TensorFlow binary was not compiled to use: SSE4.1 SSE4.2 AVX AVX2 FMA
2018-11-18 05:50:56.797575: I tensorflow/core/common_runtime/process_util.cc:69] Creating new thread pool with default inter op setting: 2. Tune using inter_op_parallelism_threads for best performance.
